docs/ref/settings.txt

```diff
@@ -428,6 +428,15 @@ any hyphens with underscores, and adding an ``'HTTP_'`` prefix to the name.
 For example, if your client sends a ``'X-XSRF-TOKEN'`` header, the setting
 should be ``'HTTP_X_XSRF_TOKEN'``.
 
+CSRF_TRUSTED_ORIGINS
```

Comment:
Add `.. setting:: SRF_TRUSTED_ORIGINS` above this heading. There is also a topical list of settings at the bottom of this page to add this setting to.